gunicorn/arbiter.py

```diff
@@ -494,6 +496,11 @@ def manage_workers(self):
 
     def spawn_worker(self):
         self.worker_age += 1
+        worker_id = -1
+        for i in range(self.num_workers):
```

Comment:
That's incredibly inefficient.  I would rather generate a unique ID there or maybe just hash the age with the current time when the worker is created so you don't have to iterate over all the running worker to find something unique.